tests/Tests/200__monitor_and_manage/200__metrics/201__billing_metrics.robot

```diff
@@ -69,6 +70,68 @@ Test Metric "Active_Users" On OpenShift Monitoring On Cluster Monitoring Prometh
     Should Be Equal    ${value}    2    msg=Active Users are not equal to length of list or N users
     [Teardown]    CleanUp JupyterHub For N users    list_of_usernames=${list_of_usernames}
 
+Test Metric "Active Users" On Telemeter
+    [Documentation]    Verifies the openshift metrics and telemeter shows
+    ...                the same rhods active users
+    [Tags]    ODS-1054
+    ...       Tier1
```

Comment:
is it Sanity? @jgarciao

Reply:
If the execution time is long (>15 mins, aprox) it could be moved to Tier2. If not, it will be Sanity and Tier1